libs/wingsdk/src/core/lifting.ts

```diff
@@ -219,12 +219,6 @@ export function collectLifts(
   initialObj: any,
   initialOps: Array<string>
 ): Map<any, Set<string>> {
-  if (initialOps.includes(INFLIGHT_INIT_METHOD_NAME)) {
-    throw new Error(
-      `The operation ${INFLIGHT_INIT_METHOD_NAME} is implicit and should not be requested explicitly.`
-    );
-  }
```

Comment:
I'd recommend either of these options, if it's not too much of a hassle to include in this PR:

1. Fix this line that I recently changed (the likely cause of the regression)

https://github.com/winglang/wing/blob/b6d1f04b7c7d9036a100bd63178ed6134f6525c0/libs/wingsdk/src/core/inflight.ts#L224-L226

so that we filter out the $inflight_init key when generating automatic permissions:

```
Object.keys(...).filter(x => x !== "$inflight_init")
```

This would let us keep the error message (can revert the changes to lifting.ts)

2. Or, just filter out "$inflight_init" from `ops` before we call onLift() or onLiftType() here:

https://github.com/winglang/wing/blob/b6d1f04b7c7d9036a100bd63178ed6134f6525c0/libs/wingsdk/src/core/lifting.ts#L399-L428